src/components/get-started/get-started.js

```diff
@@ -15,7 +15,7 @@ export default class GetStarted extends HTMLElement {
         </div>
 
         <div>
-          <a href="https://stackblitz.com/github/projectevergreen/greenwood-getting-started" class="${styles.buttonBlitz}">
+          <a href="https://stackblitz.com/github/projectevergreen/greenwood-getting-started" class="${styles.buttonBlitz}" target="_blank">
             <span>View in Stackblitz</span>
```

Comment:
One recommendation may be to add a hidden `<span>` to better help identify these links to screen readers that they open in a new tab, by just hidding the visual content, but will still get read aloud to users
```html
<a>
  <span>View in Stackblitz</span>
  <span style="height: 0; width: 0">Opens in a new tab</span>
</a>
```

Or better yet with a class
```css
screen_reader_text {
    clip: rect(0 0 0 0);
    clip-path: inset(50%);
    height: 1px;
    overflow: hidden;
    position: absolute;
    white-space: nowrap;
    width: 1px;
}
```